irohad/ametsuchi/impl/in_memory_storage.hpp

```diff
@@ -0,0 +1,92 @@
+/**
```

Comment:
I think we introduced that for the purpose of testing different assumptions. Do we need to keep that? if so, we should write some comment saying that this implementation is for testing purposes only